### PR TITLE
Runtime Tests

### PR DESCRIPTION
**Pull request by Xiaoy312**
GitHub Issue (If applicable): #356

## PR Type

What kind of change does this PR introduce?
- Project automation

## What is the new behavior?
Runtime Tests

## PR Checklist
Please check if your PR fulfills the following requirements:
- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by jeromelaban — APPROVED** [review]

**Comment by Xiaoy312** [comment]
todo:
- [ ] await unoplatform/uno.ui.runtimetests.engine#14
- [ ] bump that package

**Comment by jeromelaban** [comment]
For the [build error you're getting](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=59407&view=logs&j=366b0b7a-f786-59f1-8b55-17c16ce5937b&t=6da899b3-6ac1-51c2-b704-506dddd482ac&l=35232), it's likely that [the workaround for xbfs ](https://github.com/unoplatform/uno.toolkit.ui/blob/main/src/winappsdk-workaround.targets)is not being applied properly.

File: src/Uno.Toolkit.RuntimeTests/Uno.Toolkit.RuntimeTests.props [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		Condition="'$(FrameworkLineage)' == 'WinUI'"
+	-->
+
+	<PropertyGroup>
+		<!-- We are not building any target in UnoTargetFrameworkOverride, but only those that are supported and in the appropriate linage -->
+		<!-- condition: add target if no override or the target is defined in override -->
+		<TargetFrameworks Condition="'$(UnoTargetFrameworkOverride)'=='' OR $(UnoTargetFrameworkOverride.Contains('netstandard2.0'))">$(TargetFrameworks);netstandard2.0</TargetFrameworks>
+		<TargetFrameworks Condition="'$(UnoTargetFrameworkOverride)'=='' OR $(UnoTargetFrameworkOverride.Contains('xamarinios10'))">$(TargetFrameworks);xamarinios10</TargetFrameworks>
+		<TargetFrameworks Condition="'$(UnoTargetFrameworkOverride)'=='' OR $(UnoTargetFrameworkOverride.Contains('xamarinmac20'))">$(TargetFrameworks);xamarinmac20</TargetFrameworks>
+		<TargetFrameworks Condition="'$(UnoTargetFrameworkOverride)'=='' OR $(UnoTargetFrameworkOverride.Contains('MonoAndroid11.0'))">$(TargetFrameworks);MonoAndroid11.0</TargetFrameworks>
```
Comment (kazo0) [inline src/Uno.Toolkit.RuntimeTests/Uno.Toolkit.RuntimeTests.props:14]: Should probably be bumped to MonoAndroid12.0

**Review by kazo0 — APPROVED** [review]

**Comment by jeromelaban** [comment]
The issue comes from here:
image

The WinUI Build is building only the runtimetests UWP project, and not its dependencies, causing this issue.

Will fix.

**Pull request merged**